cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -887,6 +887,27 @@ COUNT { (n) WHERE n.foo = "bar" }
 a|
 New expression which returns the number of results of a subquery.
 
+a|
+label:functionality[]
+label:added[]
+[source, cypher, role="noheader"]
+----
+CREATE DATABASE ... TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
+----
+a|
+New sub-clause for `CREATE DATABASE`, which allows modifying the number of servers hosting a database in cluster environments.
```

Comment:
Should this say that `CREATE DATABASE` allows the creation of a database instead of modifying it? The same text was used for `ALTER DATABASE`, so that's why I am asking :)